src/Raven.Server/RavenServer.cs

```diff
@@ -119,7 +121,10 @@ public void Initialize()
                 _logger.Info(string.Format("Server store started took {0:#,#;;0} ms", sp.ElapsedMilliseconds));
 
             sp.Restart();
-
+            if (Pipe != null)
+            {
```

Comment:
Add comment explaining why this is.
This is the second time I'm seeing this code and going `HuGH?`